app/src/main/java/de/westnordost/streetcomplete/settings/SettingsFragment.java

```diff
@@ -57,6 +96,21 @@ public void onStart()
 		getActivity().setTitle(R.string.action_settings);
 	}
 
+	private Point getShownTile()
+	{
+		SharedPreferences preferences = getActivity().getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE);
```

Comment:
I'd prefer to have the static const here fully qualified, i.e. MapFragment.PREF_NAME etc.